lab_2_keywords_cooccurrence/main.py

```diff
@@ -17,7 +29,11 @@ def extract_phrases(text: str) -> Optional[Sequence[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not type_check(text, str):
+        return None
+    punctuation = r"[–—!¡\"“”#$%&'()⟨⟩«»*+,./:;‹›<=>?¿@\]\[\\_`{|}~…⋯-]+"
+    return [clean for phrase in rsplit(''.join((punctuation, r"(?=[$\s])|(?!=[\w\d])", punctuation)), text)
```

Comment:
overlong and incomprehensible line
+ import re and use as re.rsplit

Reply:
Fixed